doc/user_manual/hashlist.md

```diff
@@ -0,0 +1,124 @@
+5. **Hash source**: Select one of the following hash source types. 
+6. **Providing the hash**: The last field of the form will automatically adapt depending on the chosen source type. You’ll be asked to provide additional details:
+    - **Paste**: Copy and paste the hashes directly into the "Input" field.
+    - **Upload**: Select a file containing the hashes from your computer.
+    - **URL Download**: Provide a URL to download the hashlist.
+    - **Import**: This option can be used as a workaround in case of upload errors with the first version of the user interface. To import a file, first copy it to the import folder as described in the section Import a new file.
+7. **Access Group**: Modify the access group associated with the hashlist if needed.
+8. **Create Hashlist**: Click "Create Hashlist" to finalize the process. This will open a new page displaying the details of your newly created hashlist.
+
+
+## Hashlists In-Depth
+
+### Hashlists View
+Ordered by ID by default. It reports the hashlists created. A tick is accolated to the name of the hashlists if all the passwords have been retrieved. It shows the number of retrieved passwords as well as the total number of hashes. It allows to import *pre-cracked hashes* or export the retrieved passwords (*see below for more details*). The hashlists can also be archived or deleted.
+
+### Hashlists Details
+If you click on a Hashlist, either in the hashlists view, in the Tasks overview or inside a task, it brings you to the corresponding Hashlist details page. 
+
+Appart from the parameters specific to this hashlist (i.e. ID, Access Group, Hashlist name, ...), the page displays some information about the total number of hashes, the number of cracked ones and the number of remaining ones to be recovered. Clicking on one of these three values will open a new window displaying information about the Hashes of the Hashlist as detailed below.
+
+#### Hashes of Hashlist X 
+This page list all the hashes from the related hashlist. Filters can be applied to show either the cracked, the uncracked or all the hashes. According to the display filter selected, the Hashes only, the plaintext only or both are displayed. Additionnally, the cracking position (**to be defined**) can be displayed next to the cracked ones. Only 1000 hashes can be displayed at a time within a page but the user can navigate through the pages. The number of hashes per page can be configured in *Config > UI* settings.
+
+A HEX converter is present at the bottom of the page to convert any HEX values. This can be useful when the reported password is stored in a HEX format.
+
+#### Actions on the hashlist
+Several actions are offered to the user which are detailed below. Note that some of the options are logically not available if no password have been retrieved for the specific hashlist. 
+
+- **Download Report**: **will we still have this function**
```

Comment:
I think this will become legacy.. especially as this will probably later included in projects